test/src/main/scala/scala/test/twitter_scrooge/prefix_test/e/f/b/c/d/BUILD

```diff
@@ -0,0 +1,8 @@
+load("//thrift:thrift.bzl", "thrift_library")
+
+thrift_library(
+  name = "d",
+  srcs = ["B.thrift"],
+  absolute_prefixes = ["prefix_test/a", "prefix_test/e/f"],
```

Comment:
We have the list because I wanted to ensure the list works to test the functionality.

Maybe, but deps on thrift don't do anything? So yes, but doesn't matter too much i guess?